test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.client.cs

```diff
@@ -98,7 +98,8 @@ public void GrpcClientCallsAreCollectedSuccessfully(string baseAddress, bool sho
 
             // Tags added by the library then removed from the instrumentation
             Assert.Null(activity.GetTagValue(GrpcTagHelper.GrpcMethodTagName));
-            Assert.NotNull(activity.GetTagValue(GrpcTagHelper.GrpcStatusCodeTagName));
+            Assert.Null(activity.GetTagValue(GrpcTagHelper.GrpcStatusCodeTagName));
+            Assert.NotNull(activity.GetTagValue(SemanticConventions.AttributeRpcGrpcStatusCode));
```

Comment:
Minor thing, though should be able to make this an `Assert.Equal(0, activity.GetTagValue(SemanticConventions.AttributeRpcGrpcStatusCode))` (I _think_ this test should result in a status code of 0).

Reply:
just updated!